libnmstate/nm/bridge_port_vlan.py

```diff
@@ -18,18 +18,62 @@
 #
 
 
+from libnmstate.nm import nmclient
+from libnmstate.schema import LinuxBridge as LB
+
+
 class PortVlanFilter(object):
-    def __init__(self, trunk_tags, tag, enable_native_vlan):
-        self._trunk_tags = trunk_tags
+    pvid = "pvid"
+    min_range = "vlan-range-min"
+    max_range = "vlan-range-max"
+    untagged = "untagged"
+
+    def __init__(self, trunk_tags, tag, is_native_vlan=False):
```

Comment:
I commented about this earlier, but it may fit here as well:
This object is created from the state data but some of its static methods are implementing the other direction.
This means that this object is not a proxy that can translate in both directions and the static methods do not really belong to it.
If you want the object to be a two way lane, then it needs to input both options (on other objects we used `create` for data from state and `import` for filling it from existing NM setting/profile).
The alternative is to have it represent one direction and another using a function or another class.

Reply:
Please point me to references where I can follow the current status quo of the project, since it eludes me.

Reply:
Hope now it's better.

Please re-open if not.